doc/po/afp_lantest.1.md.ja.po

```diff
@@ -6,7 +6,7 @@
 msgid ""
 msgstr ""
 "Project-Id-Version: Netatalk 4.3.0\n"
-"POT-Creation-Date: 2025-08-05 13:58+0200\n"
+"POT-Creation-Date: 2025-08-13 22:14+1000\n"
```

Comment:
I think I prefer not having the po file changes included in the PRs. It results in large and noisy diffs, and risks getting us stuck in merge conflict hell again.

Reply:
Right, I propose that we let po files drift during phases of active development. Then right before taggin a stable release, merge the po files and update the Japanese translation. I can formalize this in the developer documentation.